AmplifyPlugins/API/Sources/AWSAPIPlugin/AppSyncRealTimeClient/AppSyncRealTimeClient.swift

```diff
@@ -425,24 +430,37 @@ extension AppSyncRealTimeClient {
         }
     }
 
-    private func monitorHeartBeats(_ connectionAck: JSONValue?) {
+    internal func monitorHeartBeats(_ connectionAck: JSONValue?) {
```

Comment:
nit: can provide omit `internal` as that is the default when no access level is specified